test/test-v8debugapi.js

```diff
@@ -80,6 +80,16 @@ describe('v8debugapi', function() {
       });
     });
 
+  it('should disambiguate incorrect path if filename is unique',
+    function(done) {
+      require('./fixtures/foo.js');
+      var bp = { id: 0, location: {line: 1, path: '/test/foo.js'}};
+      api.set(bp, function(err) {
+        assert.ifError(err);
+        api.clear(bp);
+        done();
+      });
+    });
```

Comment:
Add another test that exercises the for loop in `findScripts`. Basically, have multiple `foo.js` files in different directories.